src/automations/checkInactivity.js

```diff
@@ -102,25 +107,29 @@ async function scrapeInactiveIssues(client, references, issues) {
   const iterator = issues[Symbol.iterator]();
 
   for (let issue of iterator) {
+    const issueNumber = issue.number;
+    const repoName = issue.repository.name;
+    const repoOwner = issue.repository.owner.login;
+    const issueTag = `${repoName}/${issueNumber}`;
+
     const inactiveLabel = issue.labels.find(label => {
       return label.name === client.cfg.activity.inactive;
     });
     if (inactiveLabel) continue;
+    if (!references.get(issueTag).review) continue;
 
     let time = Date.parse(issue.updated_at);
-    const issueNumber = issue.number;
-    const repoName = issue.repository.name;
-    const repoOwner = issue.repository.owner.login;
-    const issueTag = `${repoName}/${issueNumber}`;
-    const repoTag = issue.repository.full_name;
+    const pullReqUpdateTime = references.get(issueTag).time;
 
-    if (time < references.get(issueTag)) time = references.get(issueTag);
+    if (time < pullReqUpdateTime) time = pullReqUpdateTime;
 
     const active = client.cfg.activity.check.repositories.includes(repoTag);
 
     if (time + ms >= Date.now() || !active) continue;
 
-    const aString = issue.assignees.map(assignee => assignee.login).join(", @");
+    const assignedTo = issue.assignees;
```

Comment:
Why was this changed?

Reply:
Similar reason, to successfully pass the 80 character limit test. I made a bunch of changes, seems like i couldn't revert all of them. Thanks for pointing these out :D